lib/release.sh

```diff
@@ -110,15 +113,21 @@ if [ ! "$STAGE" == "" ]; then
       # if target stage is production, release the package version
       if [ "$STAGE" == "PROD" ]; then
       
+        # get package version id (05i)
+        CMD="sfdx force:package2:version:list --json | jq '.result[] | select((.SubscriberPackageVersionId) == \"$SFDX_PACKAGE_VERSION_ID\")' | jq -r .Id"
+        debug "CMD: $CMD"
+        SFDX_PACKAGE_ID=$(eval $CMD)
+        debug "SFDX_PACKAGE_ID: $SFDX_PACKAGE_ID"
+      
         log "Set package version as released ..."
 
-        invokeCmd "sfdx force:package:update -i \"$SFDX_PACKAGE_VERSION_ID\" --noprompt --setasreleased"
-
-      fi
+        invokeCmd "sfdx force:package2:version:update -i \"$SFDX_PACKAGE_ID\" --noprompt --setasreleased"
```

Comment:
@DouglasCAyers Does it really use `$SFDX_PACKAGE_ID`?

Reply:
Okay, I'm a bit confused. Is `$SFDX_PACKAGE_VERSION_ID` not actually the package version (starting with `05i`)?

Reply:
Ah, I see. It's the `04t`. These package IDs are so confusing.